tracing/src/main/java/com/palantir/tracing/DetachedSpan.java

```diff
@@ -86,6 +86,23 @@ static DetachedSpan start(
         return Tracer.detachInternal(observability, traceId, forUserAgent, parentSpanId, operation, type);
     }
 
+    /**
+     * Marks the beginning of a span, which you can {@link #complete} on any other thread.
+     *
+     * @see DetachedSpan#start(String)
+     */
+    @CheckReturnValue
+    static DetachedSpan start(
```

Comment:
This method isn't public and isn't used anywhere internally, so this change doesn't really do anything.